tfhe/docs/guides/strings.md

```diff
@@ -0,0 +1,102 @@
+# Strings
+
+This document explains the string type provided by the High-level API.
+
+{% hint style="info" %}
+The support for strings is based on the winning submission to a [bounty season.](https://github.com/zama-ai/bounty-program/issues/80)
+{% endhint %}
+
+**TFHE-rs** has supports for **ASCII** strings with the type `FheAsciiString`.
+
+{% hint style="info" %}
+You can enable this feature using the flag: `--features=strings` when building **TFHE-rs**.
+{% endhint %}
+
+{% hint style="warning" %}
+Strings are not yet compatible with CompactCiphertextList and CompressedCiphertextList
+{% endhint %}
+
+A lot of common operations are supported, to list a few of them:
+
```

Comment:
if the op names are the same between HL traits and the strings module let's think of linking to docs.rs

which makes me think we should update the Cargo.toml for the docs.rs build to also use the strings feature, seems we don't build strings on docs.rs at the moment